plugins/always-seekable-body-plugin.rst

```diff
@@ -0,0 +1,25 @@
+Always Seekable Body Plugin
+===========================
+
+The ``AlwaysSeekableBodyPlugin`` ensure that body of the request and body of the response are always seekable.
+This allows a lot of components that read the stream to rewind it in order to be used later by another component::
+
+    use Http\Discovery\HttpClientDiscovery;
+    use Http\Client\Common\PluginClient;
+    use Http\Client\Common\Plugin\AlwaysSeekableBodyPlugin;
+
+    $options = [
+        'use_file_buffer' => true,
+        'memory_buffer_size' => 2097152,
+    ];
+    $alwaysSeekableBodyPlugin = new AlwaysSeekableBodyPlugin($options);
+
+    $pluginClient = new PluginClient(
+        HttpClientDiscovery::find(),
+        [$alwaysSeekableBodyPlugin]
+    );
+
+The plugin supports the following options:
```

Comment:
as those are options to the BufferedStream, can we (additionally) reference the doc of that stream, if it exists? or otherwise just mention that they are passed to the BufferedStream constructor?